public/data/vasantisuthar.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "VASANTI SUTHAR",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/vasantisuthar.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/vasantisuthar",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vasanti_suthar",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Showwcase",
+      "url": "https://www.showwcase.com/vasanti-suthar",
+      "icon": "https://svgshare.com/s/dkX"
```

Comment:
You can switch this back to showwcase, we have a default globe icon that will show up instead until the icon is added but then you won’t have to update your profile when the icon is updated.

Reply:
Ok so, should I switch back to default globe icon like it was before

Reply:
@vasantisuthar update the icon to the following i think:

"icon": "showcase"

Reply:
> @vasantisuthar update the icon to the following i think:
> 
> "icon": "showcase"

There's no Showwcase icon. Also, adding random image/SVG links won't work.

Reply:
Thanks, mark this as resolved @vasantisuthar

Reply:
@vasantisuthar You can request a re-review from @emmalearnscode As this PR is being marked as "Changes Requested" still.